changelog/5928.bugfix.rst

```diff
@@ -0,0 +1 @@
+Report ``PytestUnknownMarkWarning`` at the level of the user's code, not ``pytest``'s.
```

Comment:
This PR is about all warnings, so this should probably be:

```rst
Fixed some warning reports produced by pytest to point to the correct location of the warning in the user's code.
```

Reply:
I will adapt this, was merged in from https://github.com/pytest-dev/pytest/pull/5929...

Reply:
Just needs this update in order to merge, IMO.

Reply:
I will update this when i get back home... But i will create another changelog entry for #4445 so that both are addressed